internal/dot/graph.go

```diff
@@ -20,34 +20,27 @@
 
 package dot
 
+// Ctor encodes the edges that connects results to params in a graph. It includes information
+// like the name of the constructor, the package, file, line where the constructor is
+// provided, and the params and results of the constructor.
+type Ctor struct {
```

Comment:
nit: Docs don't need to repeat obvious code. Don't need to list fields in the
docs. Maybe something along the lines of this?

```
// Ctor encodes a constructor provided to the container for the DOT graph.
```